README.md

```diff
@@ -29,6 +29,7 @@ Besides the device being present (connected to the router), the following attrib
 | interface            | main, guest                      |
 | down_kilobytes_per_s | 100                              |
 | up_kilobytes_per_s   | 100                              |
+| deco_devic           | living_room                      |
```

Comment:
Looks like you're missing an e, should be "deco_device" instead of "deco_devic"

Reply:
Should be done with the last two commits